src/os/linux/local/mode/lvm.pm

```diff
@@ -0,0 +1,268 @@
+                    { template => '%d', min => 0, max => 'total', unit => 'B', cast_int => 1, label_extra_instance => 1 }
+                ]
+            }
+        },
+        { label => 'vg-space-usage-free', nlabel => 'vg.space.free.bytes', display_ok => 0, set => {
+                key_values => [ { name => 'free' }, { name => 'used' }, { name => 'prct_used' }, { name => 'prct_free' }, { name => 'total' } ],
+                closure_custom_output => $self->can('custom_space_usage_output'),
+                perfdatas => [
+                    { template => '%d', min => 0, max => 'total', unit => 'B', cast_int => 1, label_extra_instance => 1 }
+                ]
+            }
+        },
+        { label => 'vg-space-usage-prct', nlabel => 'vg.space.usage.percentage', display_ok => 0, set => {
+                key_values => [ { name => 'prct_used' }, { name => 'used' }, { name => 'free' }, { name => 'prct_free' }, { name => 'total' } ],
+                closure_custom_output => $self->can('custom_space_usage_output'),
+                perfdatas => [
+                    { template => '%.2f', min => 0, max => 100, unit => '%', label_extra_instance => 1 }
+                ]
+            }
+        }
+    ];
+}
+
+sub new {
+    my ($class, %options) = @_;
+    my $self = $class->SUPER::new(package => __PACKAGE__, %options, force_new_perfdata => 1);
+    bless $self, $class;
+
+    $options{options}->add_options(arguments => {
+        'filter-lv:s'       => { name => 'filter_lv' },
+        'filter-vg:s'       => { name => 'filter_vg' }
+    });
+
+    return $self;
+}
+
+sub manage_selection {
+    my ($self, %options) = @_;
+
+    my ($stdout, $exit_code) = $options{custom}->execute_command(
+        command => 'lvs',
+        command_options => '--separator=, 2>&1',
+        no_quit => 1
+    );
+
+    $self->{global} = { lv_detected => 0, vg_detected => 0 };
+    $self->{dlv} = {};
+    #  LV,VG,Attr,LSize,Pool,Origin,Data%,Meta%,Move,Log,Cpy%Sync,Convert
+    #  thinpool,docker,twi-aot---,71.25g,,,1.95,0.06,,,,
+    #  lv_controlm,vg_sys,-wi-ao----,5.00g,,,,,,,,
+    my @lines = split(/\n/, $stdout);
+    shift @lines;
+    my $i = 0;
+    foreach my $line (@lines) {
+        my @fields = split(/,/, $line);
+        next if (!defined($fields[6]) || $fields[6] !~ /[0-9]/);
+        my ($vg, $lv, $data, $meta) = (centreon::plugins::misc::trim($fields[1]), centreon::plugins::misc::trim($fields[0]), $fields[6], $fields[7]);
+
+        next if (defined($self->{option_results}->{filter_lv}) && $self->{option_results}->{filter_lv} ne '' &&
+            $lv !~ /$self->{option_results}->{filter_lv}/);
+        next if (defined($self->{option_results}->{filter_vg}) && $self->{option_results}->{filter_vg} ne '' &&
+            $vg !~ /$self->{option_results}->{filter_vg}/);
+
+        $self->{lv_detected}++;
+        $self->{dlv}->{$i} = { lvName => $lv, vgName => $vg, data => $data, meta => $meta };
+        $i++;
+    }
+
```

Comment:
Add empty filters message for user :

``` if (defined($self->{option_results}->{filter_vg} && scalar(keys %{$self->{dlv}}) <= 0) {
        $self->{output}->add_option_msg(short_msg => "No volume group found.");
        $self->{output}->option_exit();
    }
    if (defined($self->{option_results}->{filter_lv} && scalar(keys %{$self->{dlv}}) <= 0) {
        $self->{output}->add_option_msg(short_msg => "No logical group found.");
        $self->{output}->option_exit();
    }
```

But I'm not sure about the behaviour if the two filters are used together. If it may have sense, maybe we should use a flag to check if each of them is empty or not.